extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -153,4 +153,14 @@ private LDAPGuacamoleProperties() {}
 
     };
 
+    /**
+     * The behavior of alias dereferncing for the LDAP connections.
```

Comment:
Trusty VIM spell checker failed me...oh, wait, nevermind.  Even if it was there, it wouldn't like dereferencing, anyway.  Fixed.